packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/FullscreenButton/FullscreenButton.js

```diff
@@ -9,16 +9,17 @@ import Button from '../Button'
 
 counterpart.registerTranslations('en', en)
 
-function FullscreenButton ({ active, onClick }) {
+function FullscreenButton ({ active, disabled, onClick }) {
   const icon = (active) ? actualSizeIcon : fullscreenIcon
   const labelKey = active ? 'actualSize' : 'fullscreen'
   const label = counterpart(`FullscreenButton.ariaLabel.${labelKey}`)
 
   return (
     <Button
       active={active}
+      disabled={disabled}
       aria-label={label}
-      onClick={onClick}
+      onClick={disabled ? Function.prototype : onClick}
```

Comment:
Disabled buttons shouldn't actually register events. I don't think the browser puts them in the tab order either.

Reply:
This might a remnant from how Grommet used to disable buttons? They used to use having an undefined onClicked handler as a proxy for disabling the button. You should be able to just set the onClick handler to the expected function and then use the `disabled` prop directly for managing disabled state now just like the browser does it.

Reply:
Cool - I've removed these checks.